bridge/engine/ActiveMediaList.cpp

```diff
@@ -280,13 +284,25 @@ void ActiveMediaList::updateLevels(const uint64_t timestampMs)
 
 // recently unmuted participants have some advantage because the score is higher as the
 // noise level is likely lower than unmuted.
-size_t ActiveMediaList::rankSpeakers(float& currentDominantSpeakerScore)
+size_t ActiveMediaList::rankSpeakers(float& currentDominantSpeakerScore,
+    TActiveTalkersSnapshot& outActiveTalkersSnapshot)
 {
     currentDominantSpeakerScore = 0;
 
     size_t speakerCount = 0;
     for (auto& participantLevelEntry : _audioParticipants)
     {
+        if (_c9_conference)
```

Comment:
Perhaps && is better than nesting in this case.

Reply:
agree